backend/src/packages/auth/auth.service.ts

```diff
@@ -18,6 +25,33 @@ class AuthService {
     return this.userService.create(userRequestDto);
   }
 
+  public async verifyLoginCredentials({
+    email,
+    password,
+  }: UserSignInRequestDto): Promise<UserSignInResponseDto | null> {
+    const user = await this.userService.findByEmail(email);
+
+    if (!user) {
+      throw new UserNotFoundError({
```

Comment:
We will use error per service, not per status or specific error. So instead of UserNotFoundError and IncorrectPasswordError, you should have UsersError and pass needed status code

Reply:
Done!